src/utils/containerCode.ts

```diff
@@ -28,8 +27,11 @@ const createRequestHandler: RuntimeRequestHandler = async (
  * By including the createRequestHandler, we can create any droplet types we include in the registry on-demand.
  * These can then be retrieved via container.request("/dataObjectId").
  */
-export const KeyValueContainerRuntimeFactory = new BaseContainerRuntimeFactory(
-    [KeyValueInstantiationFactory.registryEntry],
-    [],
-    [createRequestHandler, innerRequestHandler]
-);
+export const getRuntimeFactory = (factories: any[]) => {
```

Comment:
Maybe DataObjectFactory rather than any?

Reply:
Yeah, wasn't sure which interface..i'll try that

Reply:
![image](https://user-images.githubusercontent.com/1434956/106672411-d800d400-6564-11eb-864f-cd64158bce96.png)

Yeah, i had tried that. DataObjectFactory has a bunch of generics

Reply:
Oof, ok - maybe something to work on later then.